connectivity/check/deployment.go

```diff
@@ -257,7 +257,7 @@ func newDaemonSet(p daemonSetParameters) *appsv1.DaemonSet {
 							ReadinessProbe:  p.ReadinessProbe,
 							SecurityContext: &corev1.SecurityContext{
 								Capabilities: &corev1.Capabilities{
-									Add: []corev1.Capability{"NET_RAW"},
+									Add: []corev1.Capability{"NET_ADMIN", "NET_RAW"},
```

Comment:
Will this affect all connectivity tests any user runs in their environment? It seems to me that "configuring routes on the node" is very much outside the realm of stuff that `cilium connectivity test` should do in a production environment :thinking:

Reply:
Synced offline. Moved to the CI infra / workflow logic to install static routes on the infrastructure rather than rely on the CLI to do it.